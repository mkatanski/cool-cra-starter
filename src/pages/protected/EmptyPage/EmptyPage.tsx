import React from "react";

const EmptyPage: React.FC = () => <h1>Empty Page</h1>;

export default EmptyPage;
