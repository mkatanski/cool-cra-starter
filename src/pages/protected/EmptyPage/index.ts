import EmptyPage from "./EmptyPage";
export * from "./EmptyPage";

export default EmptyPage;
