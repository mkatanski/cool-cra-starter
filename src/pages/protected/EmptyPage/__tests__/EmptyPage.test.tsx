/* eslint-disable react/display-name */
import React from "react";
import TestRenderer from "react-test-renderer";

import EmptyPage from "../EmptyPage";

describe("pages", () => {
  describe("protected", () => {
    describe("EmptyPage", () => {
      it("should render EmptyPage component", () => {
        const tree = TestRenderer.create(<EmptyPage />);
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
