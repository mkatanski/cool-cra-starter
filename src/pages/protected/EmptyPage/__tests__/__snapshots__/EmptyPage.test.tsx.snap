// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages protected EmptyPage should render EmptyPage component 1`] = `
<h1>
  Empty Page
</h1>
`;
